templates/ts-apollo-runtime-backend/src/utils.ts

```diff
@@ -0,0 +1,36 @@
+import { loadConfigSync } from 'graphql-config';
+import Knex from 'knex';
+
+export function getProjectConfig() {
```

Comment:
If this will be moved to cli then utils file can be removed

Reply:
Can you explain why it should be in the CLI or how that would work? Do you mean core?

Reply:
We will need it somewhere but what we have currently is graphql-config only on the cli. Moving this code anywhere will mean moving it to CLI as CLI should be imported into a template anyway to execute generate. Moving to core will mean bringing some dependencies there that are not relevant for the rest but leaving decision for yourself. Generally WDYT about this idea?

Reply:
CLI should only be a devDependency, so it shouldn't be used in the template code though, right?